src/HotChocolate/Core/src/Types/SchemaOptions.cs

```diff
@@ -15,58 +15,24 @@ public class SchemaOptions : IReadOnlySchemaOptions
     private BindingBehavior _defaultBindingBehavior = BindingBehavior.Implicit;
     private FieldBindingFlags _defaultFieldBindingFlags = FieldBindingFlags.Instance;
 
-    /// <summary>
```

Comment:
why did you remove the xml docs?

Reply:
It's not needed. It's provided through the underlying interface. This way the descriptions only need to be maintained on the interface.

Reply:
All public types must have xml docs even with interface. But you can add an inheritance xml tag

Reply:
Where does this must come from? Is it convention of this code base, because Rider and VS Code don't require it.

Reply:
I've added inheritdoc regardless for now...